### ISSUE TITLE
Did this ever work?

### ISSUE THREAD

**Issue by BernardoGiordano** [Slluxx/serverxbrowser#1]
I've been trying to do something similar, but I couldn't connect to localhost. I have the same issue running your software. Did this ever work for you?

**Comment by Slluxx** [comment]
Yeah somewhat. It worked once after some tinkering with atmosphere, the Browsers RomFS and DNS_MITM. The problem is that (apparently) the switch does not have a loopback address. TBH its not worth it, as you have to modify too many things which endusers wont like and i cant for my life remember what specifically it was. 

If you want to transport data to the browser, you can do so via the launch url (eg: myurl.com?name=value) and you can grab the last url the browser has visited and parse the data from it the same way. However you are limited to 1000 characters.

If you want to transmit/receive data while the browser is running, you can use [`webSessionTrySendContentMessage`](https://switchbrew.github.io/libnx/web_8h.html#afced762743a00750d8725f562637528f) and [`webSessionTryReceiveContentMessage`](https://switchbrew.github.io/libnx/web_8h.html#ac224bfe0869997c5704d7b8f3055177f), however i have never tried it because the browser halts the thread so this seemingly has to be used in a different one.

**Comment by BernardoGiordano** [comment]
Thanks for your response. Would you still be able to check the DNS MITM configuration you have on your atmosphere hosts file?

**Comment by Slluxx** [comment]
> Thanks for your response. Would you still be able to check the DNS MITM configuration you have on your atmosphere hosts file?

sorry, since then i have reinstalled atmosphere countless times for other projects, so that knowledge is long gone
